packages/@atjson/source-html/src/converter/paragraphs.ts

```diff
@@ -0,0 +1,33 @@
+import Document from "@atjson/document";
+import { Paragraph } from "@atjson/offset-annotations";
+import { parseCSS, toAlignment } from "./utils";
+
+export default function (doc: Document) {
+  doc
+    .where({ type: "-html-p" })
+    .as("paragraph")
+    .outerJoin(
+      doc
+        .where((annotation) => annotation.attributes.lang != null)
```

Comment:
I think we should be using the `dir` attribute to determine text direction, not the `lang`. The W3 recommendation is not to assume that language indicates directionality or vice versa: https://www.w3.org/International/questions/qa-html-dir

Reply:
I think I'm going to detect `rtl` explicitly and throw an error if we encounter it, since I don't think we still have a bunch of work to do before we support bidi languages